tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -106,6 +106,13 @@ var _ = Describe("odo devfile push command tests", func() {
 			helper.CmdShouldPass("odo", "push", "--project", namespace)
 		})
 
+		It("checks that odo push works with a devfile with now flag", func() {
+			context1 := helper.CreateNewContext()
```

Comment:
Why we are creating a temp context dir again? I can see one created in BeforeEach() https://github.com/openshift/odo/pull/3496/files#diff-c127bd883f89dd3a7abff6183ad9921dR26

Reply:
will do the update , thanks